sdk/core/azure-core-perf/src/main/java/com/azure/core/perf/XmlSendTest.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.perf;
+
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.rest.RestProxy;
+import com.azure.core.perf.core.MockHttpReceiveClient;
+import com.azure.core.perf.core.MyRestProxyService;
+import com.azure.core.perf.core.RestProxyTestBase;
+import com.azure.core.perf.core.TestDataFactory;
+import com.azure.core.perf.models.UserDatabase;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Mono;
+
+public class XmlSendTest extends RestProxyTestBase<PerfStressOptions> {
+    private final MockHttpReceiveClient mockHttpReceiveClient;
```

Comment:
Might be worth investigating how to run all these tests in such a way that they end up talking over a socket. Either with test proxy or [wiremock](https://wiremock.org/). This could be an option when running the test.